example/main.tf

```diff
@@ -24,7 +25,7 @@ module "postgresql" {
 # only for use when building from ADO and as a quick example to get valid tags
 # if you are building from Jenkins use `var.common_tags` provided by the pipeline
 module "common_tags" {
-  source = "git@github.com:hmcts/terraform-module-common-tags?ref=master"
+  source = "github.com/hmcts/terraform-module-common-tags?ref=master"
```

Comment:
any reason to change this? (I suggest reverting)

Reply:
Failing to be checked out via the git@ reference

Reply:
may need to just do an `ssh-keyscan github.com`

Reply:
Still fails:
https://github.com/hmcts/terraform-module-postgresql-flexible/runs/6667376414?check_suite_focus=true#step:4:11
https://github.com/hmcts/terraform-module-postgresql-flexible/runs/6667376414?check_suite_focus=true#step:6:19